bin/process_cft_data.py

```diff
@@ -32,7 +32,10 @@ def h(*args, **kw_args):
     return h
 
 def strip_ns(a):
-    return a.split(':')[-1]
+    after_colon_name = a.split(':')[-1]
+    if after_colon_name.startswith('_'):
+        return a.split(':')[-2].split('.')[-1] + 's'
```

Comment:
Not sure what's going on here.

Reply:
A hack to make reverse lookups plural in name.

Reply:
@metasoarous I can write a comment to explain? Or do you think it needs to be done differently?

Reply:
Ooohhh... I'd actually recommend against this. Consider the fishs.

Reply:
Actually... thinking about this a bit more, maybe we're doing this in a limited enough capacity in this case that we're okay. Since this is just for CFT data, it might be alright.

Reply:
Yeah, it definitely is not scalable / flexible. Some alternatives are adding plural names for certain keys on an individual basis or just not having plural names at all (or all plural) which would be ideal and probably most flexible.

Reply:
We have more or less already committed to the spec going into Olmsted as having pluralization where appropriate, so we need to do this one way or the other. With that in mind, we may as well try to be general about how we do it. This is maybe a good way forward: https://pypi.org/project/inflect/